src/App.css

```diff
@@ -1,30 +1,36 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;500&family=Raleway:wght@200;400&display=swap');
+
 * {
     box-sizing: border-box;
     margin: 0;
+    font-family: "Poppins", sans-serif;
   }
 
-  .header {
-    background-color: rgb(107, 171, 255);
-    padding: 10px;
-    display: flex;
-    justify-content: center;
-    align-items: center;
-  }
-  
   .header-container {
     width: 100%;
     background-color: rgb(107, 171, 255);
     display: flex;
-    justify-content: space-between;
+    justify-content: center;
```

Comment:
Responsiveness:
This app looks like it was designed web first rather than mobile first. I would strongly recommend doing mobile first in the future as it is much easier from a CSS perspective to scale up from mobile to web.

When squeezing the width in of your app, the comments section doesn't respond appropriately